src/ImmersedBoundaries/ImmersedBoundaries.jl

```diff
@@ -102,26 +102,27 @@ abstract type AbstractImmersedBoundary end
 ##### ImmersedBoundaryGrid
 #####
 
-struct ImmersedBoundaryGrid{FT, TX, TY, TZ, G, I, M, Arch} <: AbstractGrid{FT, TX, TY, TZ, Arch}
+struct ImmersedBoundaryGrid{FT, TX, TY, TZ, G, I, M, S, Arch} <: AbstractGrid{FT, TX, TY, TZ, Arch}
     architecture :: Arch
     underlying_grid :: G
     immersed_boundary :: I
     interior_active_cells :: M
-    
+    surface_active_cells :: S
```

Comment:
surface, like surface of the ocean? or boundary, like boundary conditions?

Reply:
changed it to `active_z_columns`